indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -252,6 +255,37 @@ def _process(self):
                                                     regform=self.regform)
 
 
+class RHRegistrationFormNotificationPreview(RHManageRegFormBase):
+    """Preview registration emails."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.commit = False
+
+    @no_autoflush
```

Comment:
Please also double-check via the db log that we never flush this dummy registration (as that would consume one ID every time)

Reply:
Yes, I confirm there is no `INSERT` statement with that.